frontend/src/App.tsx

```diff
@@ -68,6 +69,12 @@ const App = (): React.ReactElement => {
                 <Route exact path={Routes.LOGIN_PAGE} component={Login} />
                 <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
                 <Route exact path={Routes.LANDING_PAGE} component={Home} />
+                <PrivateRoute
```

Comment:
if you rebase your branch on top of the current changes in staging, let's route to this create check in page from the button (mobile/desktop): 
https://github.com/uwblueprint/community-fridge-kw/blob/d275f4e8951ef3d4dd18e6db5c219f9f4d20ed9b/frontend/src/components/pages/AdminDashboard/components/CheckInAdminButtons.tsx#L30
https://github.com/uwblueprint/community-fridge-kw/blob/d275f4e8951ef3d4dd18e6db5c219f9f4d20ed9b/frontend/src/components/pages/AdminDashboard/components/CheckInAdminButtons.tsx#L65